colorBackground.js

```diff
@@ -0,0 +1,17 @@
+var usedColors = {};
```

Comment:
putting stuff in the global scope is bad practice, whats a nice way to avoid this?